.github/workflows/audit_build_verify.yml

```diff
@@ -0,0 +1,128 @@
+name: Kwenta CI
```

Comment:
Not sure how or why this file was deleted

Reply:
I think it's one of the merged changes from the `perps-v2-dev` branch.

Reply:
It's currently broken, hence I removed it when cleaning up #1803 ; just noticed once I'm back I need to set up all the other checks for the new `perps-v2-dev` branch everywhere; or feel free to do so beforehand.